README.md

```diff
@@ -74,6 +74,58 @@ EOF
+  ]
+}
+```
+
+`"configList"` should contain a list of config objects. Each config object may consist of following fields:
+
+
+|     Field      | Required |                                                    Description                                                                     |     Type         |                         Example                         |
+|----------------|----------|------------------------------------------------------------------------------------------------------------------------------------|------------------|---------------------------------------------------------|
+| "resourceName" | Y        | Endpoint resource name. Should not contain special characters, must be unique in the scope of the resource prefix                  | string           | "hca_shared_devices_a"                                  |
+| "rdmaHcaMax"   | Y        | Maximum number of RDMA resources that can be provided by the device plugin resource                                                | Integer          | 1000                                                    |
+| "selectors"    | N        | A map of device selectors for filtering the devices. refer to [Device Selectors](#devices-selectors) section for more information  | json object      | selectors": {"vendors": ["15b3"],"deviceIDs": ["1017"]} |
+| "devices"      | N        | A list of devices names to be selected, same as "ifNames" selector                                                                 | `string` list    | ["ib0", "ib1"]                                          |
+
+Note: Either `selectors` or `devices` must be specified for a given resource, "selectors" is recommended.
+
+## Devices Selectors
+The following selectors are used for filtering the desired devices.
+
+
+|    Field    |                          Description                           |     Type      |         Example          |
+|-------------|----------------------------------------------------------------|---------------|--------------------------|
+| "vendors"   | Target device's vendor Hex code as string                      | `string` list | "vendors": ["15b3"]      |
+| "deviceIDs" | Target device's device Hex code as string                      | `string` list | "devices": ["1017"]      |
+| "drivers"   | Target device driver names as string                           | `string` list | "drivers": ["mlx5_core"] |
+| "ifNames"   | Target device name                                             | `string` list | "ifNames": ["enp2s2f0"]  |
+| "linkTypes" | The link type of the net device associated with the PCI device | `string` list | "linkTypes": ["ether"]   |
+
+[//]: # (The tables above generated using: https://ozh.github.io/ascii-tables/)
+
+## Selectors Matching Process
+The device plugin filters the host devices based on the provided selectors, if there are any missing selectors, the device plugin ignores them. The devices that match all the provided selectors will be used by the device plugin.
```

Comment:
what happens if more than one string is provided to each selector ? 
logical *OR* is performed between elements of a specific selector and logical *AND* is performed between selectors.

you covered this in the design doc IIRC